benchmarks/nel/scripts/datasets/dataset.py

```diff
@@ -220,61 +249,86 @@ def evaluate(
         candidate_generation: bool = True,
         baseline: bool = True,
         context: bool = True,
+        spacyfishing: bool = True,
```

Comment:
What does this do exactly?

Reply:
`spacyfishing` fetches its result individually via an API. This slows down the evaluation process by a lot. For the sake of convenience (esp. with bigger eval. sets) I added this flag in case I just wanted to compare our internal implementations.